app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionContentViewModel.kt

```diff
@@ -20,5 +20,8 @@ class SelectionInteractionContentViewModel(
     if (isCurrentlySelected != shouldNowBeSelected) {
       isAnswerSelected.set(shouldNowBeSelected)
     }
+    selectionInteractionViewModel.selectedItemsList?.forEach {
```

Comment:
We need to pass in the observable boolean corresponding to this specific checkbox from SelectionInteractionContentViewModel's constructor.

Reply:
@BenHenning, How can we do that here can you show me some example?

Reply:
I'm not sure if there is an exact example @Akshatkamboj14. In this case, I think you need to think through the logic of how this works and try to come up with something that makes sense to you. I don't think this is a case were can just copy and paste from another location in code.

Reply:
Okay, @BenHenning I'll think of something.